### ISSUE TITLE
Leaflet Cluster-Hexagon overlay

### ISSUE THREAD

**Issue by nicolas-f** [Ifsttar/NoiseCapture#202]
Currently hexagons clusters and measurement hexagons are shown on the same zoom level 14. 

http://onomap.noise-planet.org/#14/47.0569/-0.8715

@Picaut @ebocher

**Comment by ebocher** [comment]
Yes, the scale denominator must be adapted when the location of measures is rendered.
It's not necessary to display both cluster of hexagons and point locations.
Try this link http://onomap.noise-planet.org/#14/45.7568/4.8234

The same about noise hexagons

**Comment by ebocher** [comment]
@Picaut  ready to adapt the scale denominator ?

**Comment by nicolas-f** [comment]
You want to keep cluster or noise level at level 14 ?

Because at level 14 we still may don't see noise levels:

http://onomap.noise-planet.org/#14/36.0082/-114.9400
